processing/src/main/java/org/apache/druid/data/input/Rows.java

```diff
@@ -70,6 +71,8 @@ public static List<String> objectToStrings(final Object inputValue)
     } else if (inputValue instanceof byte[]) {
       // convert byte[] to base64 encoded string
       return Collections.singletonList(StringUtils.encodeBase64String((byte[]) inputValue));
+    } else if (inputValue instanceof Object[]) {
+      return Arrays.stream((Object[]) inputValue).map(String::valueOf).collect(Collectors.toList());
```

Comment:
use `Evals::asString` instead of `String::valueOf` since the latter will translate `null` into `"null"` which isn't cool